tests/statsmodels/test_statsmodels_model_export.py

```diff
@@ -1,11 +1,11 @@
 import pytest
 import numpy as np
 import pandas as pd
-import mock
 import os
 import yaml
 import json
 import pandas.testing
+from unittest import mock

```

Comment:
looks like previous failures (e.g., https://github.com/mlflow/mlflow/runs/1535763935) were attributable to using `mock` via `import mock` instead of `from unittest import mock` in `test_autologging_safety_unit.py`. 
There's a documented incompatibility between `@mock.patch` when accessed via `import mock` and pytest described here: https://stackoverflow.com/questions/39169563/pytest-works-with-old-mock-but-not-unittest-mock.

For safety going forward, I've converted the only other instance of `import mock` in our statsmodels flavor tests to `from unittest import mock`.